TimePlatformer/Assets/Scripts/WindScript.cs

```diff
@@ -0,0 +1,36 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class WindScript : MonoBehaviour {
+
+	/// <summary>
+	/// the variable, which should be set externally, that denotes whether this is day wind or not.
+	/// </summary>
+	public bool day;
```

Comment:
Instead of making this a boolean, please make it to the `TimeOfDay` enumeration.